components/brave_vpn/resources/panel/components/error-panel/style.ts

```diff
@@ -2,89 +2,98 @@
-  position: relative;
-  z-index: 2;
+  align-itmes: flex-start;
+  align-self: stretch;
 `
 
-export const ReasonTitle = styled.h1`
-  color: ${(p) => p.theme.color.text01};
-  font-family: ${(p) => p.theme.fontFamily.heading};
-  font-size: 18px;
-  font-weight: 600;
-  letter-spacing: 0.02em;
-  line-height: 2.6;
-  margin: 0;
-  text-align: center;
+export const PanelHeader = styled.section`
+  display: flex;
+  padding: var(--leo-spacing-2xl) var(--leo-spacing-2xl) var(--leo-spacing-xl)
+    var(--leo-spacing-2xl);
+  align-items: center;
+  gap: var(--leo-spacing-m);
 `
 
-export const ReasonDesc = styled.p`
-  color: ${(p) => p.theme.color.text01};
-  font-family: ${(p) => p.theme.fontFamily.heading};
-  font-size: 14px;
-  font-weight: 400;
-  letter-spacing: 0.02em;
-  margin: 0;
-  text-align: center;
-  padding-bottom: 24px;
+export const StyledIcon = styled(Icon)`
+  --leo-icon-size: 20px;
+  padding: 8px 0px;
+  color: var(--leo-color-icon-default);
 `
 
-export const ActionArea = styled.div`
-  width: 100%;
-  display: grid;
+export const StyledButton = styled.button`
+  background-color: transparent;
+  border: 0;
+  width: 20px;
+  height: 20px;
+  padding: 0;
+  cursor: pointer;
+`
 
-  button {
-    &:first-child {
-      margin-bottom: 10px;
-    }
-  }
+export const HeaderLabel = styled.span`
+  color: var(--leo-color-text-primary);
+  font-family: Poppins;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 26px;
 `
 
-export const PanelHeader = styled.section`
+export const TopContent = styled.div`
   display: flex;
+  padding: var(--Spacing-None, 0px) var(--leo-spacing-xl) var(--leo-spacing-xl)
+    var(--leo-spacing-xl);
+  flex-direction: column;
   align-items: center;
-  width: 100%;
-  margin-bottom: 10px;
-  box-sizing: border-box;
+  gap: var(--leo-spacing-m);
+  align-self: stretch;
 `
 
-export const BackButton = styled.button`
-  background-color: transparent;
-  border: 0;
-  padding: 0;
+export const ErrorContent = styled.div`
   display: flex;
-  align-items: center;
-  cursor: pointer;
+  flex-direction: column;
+  padding: var(--leo-spacing-2xl);
+  margin-bottom: var(--leo-spacing-xl);
+  align-items: flex-start;
+  gap: 4px;
+  border-radius: var(--leo-radius-m);
+  background: var(--leo-color-systemfeedback-error-background);
+  align-self: stretch;
+`
 
-  span {
-    font-family: ${(p) => p.theme.fontFamily.heading};
-    font-size: 13px;
-    font-weight: 600;
-    color: ${(p) => p.theme.color.interactive05};
-  }
+export const ReasonTitle = styled.div`
+  color: var(--leo-color-systemfeedback-error-text);
+  font-family: ${(p) => p.theme.fontFamily.heading};
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 26px;
+`
 
-  i {
-    width: 18px;
-    height: 18px;
-    margin-right: 5px;
-  }
+export const ReasonDesc = styled.div`
+  color: var(--leo-color-systemfeedback-error-text);
+  font-family: "Inter Variable";
```

Comment:
@fallaciousreasoning Can you give advice what is the exact way to set this value when figma gives `font-family: var(--Family-Default, "Inter Variable");`?

Reply:
I think it should be

`font: ${leo.font.small.regular};`

![image](https://github.com/user-attachments/assets/6bcba8d1-c6b2-498a-9a04-0040bced95dd)

going off the name in Figma. Unfortunately the variable names Figma comes up with only have a tenuous relationship with reality.

Optionally, I think you could use the `Alert` component here (and slot empty content in to the icon):
https://nala.bravesoftware.com/?path=/story/components-alert--slots&args=type:warning

Looks like the padding might be a bit off though

Reply:
@fallaciousreasoning Thanks! I'll try `Alert` compoent 👍🏼

Reply:
https://github.com/brave/leo/pull/757 should fix the spacing on the alert when hiding the icon (you can set the `hideIcon` prop)